docs/get-started/Velox.md

```diff
@@ -352,7 +352,7 @@ Using the following configuration options to customize spilling:
 | spark.gluten.sql.columnar.backend.velox.spillStartPartitionBit           | 29            | The start partition bit which is used with 'spillPartitionBits' together to calculate the spilling partition number                                                               |
 | spark.gluten.sql.columnar.backend.velox.spillPartitionBits               | 2             | The number of bits used to calculate the spilling partition number. The number of spilling partitions will be power of two                                                        |
 | spark.gluten.sql.columnar.backend.velox.spillableReservationGrowthPct    | 25            | The spillable memory reservation growth percentage of the previous memory reservation size                                                                                        |
-
+| spark.gluten.sql.columnar.backend.velox.spillThreadNum                   | 0             | The thread num of a dedicated thread pool to do spill
```

Comment:
If the functionality is not yet fully tested, I think we can add an `(experimental)` annotation here?